server/methods/accounts/accounts.app-test.js

```diff
@@ -5,24 +5,21 @@ import { Meteor } from "meteor/meteor";
 import { Factory } from "meteor/dburles:factory";
 import { check, Match } from "meteor/check";
 import { Random } from "meteor/random";
-import { Accounts as MeteorAccount } from "meteor/accounts-base";
+import { Accounts as MeteorAccounts } from "meteor/accounts-base";
 import { expect } from "meteor/practicalmeteor:chai";
 import { sinon } from "meteor/practicalmeteor:sinon";
-import { Accounts, Packages, Orders, Products, Shops, Cart } from "/lib/collections";
-import { Reaction } from "/server/api";
+import { SSR } from "meteor/meteorhacks:ssr";
+import { Accounts, Groups, Packages, Orders, Products, Shops, Cart } from "/lib/collections";
+import { Logger, Reaction } from "/server/api";
 import { getShop, getAddress } from "/server/imports/fixtures/shops";
 import Fixtures from "/server/imports/fixtures";
 
 Fixtures();
 
-before(function () {
```

Comment:
I believe using the `before` block here sets the timeout globally. that does not seem necessary for most test files.